Packs/CortexXDR/Scripts/CortexXDRIdentityInformationWidget/CortexXDRIdentityInformationWidget.py

```diff
@@ -0,0 +1,66 @@
+"""Base Script for Cortex XSOAR (aka Demisto)
+
+This is an empty script with some basic structure according
+to the code conventions.
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+"""
+
+import demistomock as demisto
+from CommonServerPython import *
+
+import traceback
+
+''' COMMAND FUNCTION '''
+
+
+def get_identity_info() -> List[Dict]:
+    context = demisto.context()
+    alerts = demisto.get(context, 'PaloAltoNetworksXDR.OriginalAlert')
+    users = demisto.get(context, 'AWS.IAM.Users')
+    if not users:
+        raise DemistoException('AWS users are not in context')
+    access_keys = users[0].get('AccessKeys', [])
+    if not isinstance(alerts, list):
+        alerts = [alerts]
+    results = []
+    for alert in alerts:
+        alert_event = alert.get('event')
+        username = alert_event.get('identity_orig').get('userName')
+        access_keys_ids = list({access_key.get('AccessKeyId') for access_key in access_keys
+                                if access_key and access_key.get('UserName') == username})
```

Comment:
I assume `access_key` has to be a dict, right?
```suggestion
                                if isinstance(access_key, dict) and access_key.get('UserName') == username})
```